src/api/custom-resolvers/mutations/create-feedback.ts

```diff
@@ -0,0 +1,45 @@
+import { Context } from '.keystone/types'; // eslint-disable-line
+import { InsuranceFeedbackVariables } from '../../types';
+import sendEmail from '../../emails';
+
+/**
+ * creates feedback in database and sends email
+ * adds createdAt timestamp to feedback
+ * @param root
+ * @param {InsuranceFeedbackVariables} variables
+ * @param context
+ * @returns {Object} with success true or false and response
+ */
+const createInsuranceFeedbackAndEmail = async (root: any, variables: InsuranceFeedbackVariables, context: Context) => {
+  console.info('Creating feedback');
```

Comment:
And in the catch handler, should it be "insurance feedback"?

Reply:
keeping it generic for all feedback